src/main/scala/is/hail/check/Prop.scala

```diff
@@ -87,7 +102,11 @@ class Properties(val name: String) extends Prop {
 
   override def apply(p: Parameters, prefix: Option[String]) {
     for ((propName, prop) <- properties)
-      prop(p, prefix.map(_ + "." + propName).orElse(Some(propName)))
+      prop.apply(p, prefix.map(_ + "." + propName).orElse(Some(propName)))
+  }
+
+  def checkSingle(propName: String) {
+
```

Comment:
oops, meant to actually finish this. I want a way to run one specific method from a Spec without running them all.